.github/actions/repo-gardening/src/tasks/notify-design/index.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * External dependencies
+ */
+const { getInput } = require( '@actions/core' );
+
+/**
+ * Internal dependencies
+ */
+const debug = require( '../../debug' );
+const getLabels = require( '../../get-labels' );
+const sendSlackMessage = require( '../../send-slack-message' );
+
+/* global GitHub, WebhookPayloadPullRequest */
+
+/**
+ * Check for a Design Review status label on a PR.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Promise<boolean>} Promise resolving to boolean.
+ */
+async function hasDesignReviewLabel( octokit, owner, repo, number ) {
```

Comment:
I keep getting confused because the name of this function doesn't match the label.
```suggestion
async function hasNeedsDesignReviewLabel( octokit, owner, repo, number ) {
```